cmd/pg/daemon.go

```diff
@@ -13,8 +14,12 @@ var daemonCmd = &cobra.Command{
 	Short: DaemonShortDescription,
 	Args:  cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		storageStatusCache, err := postgres.InitStatusCache(true)
```

Comment:
I moved opening new conenctions to the place that called only once instead of closing connections for two reasons:
1. I tried to close them, but it's not so easy: we can't just close `storage.Folder`s. because multiple folders can share the same connection (e.g. all folders got by listing the original folder share the same connection). So for proper closing, we need to return something (in addition to the folder itself) from `internal.ConfigureFolder` and close it later. We still can do that, but in another issue.
2. Even with proper closing, it's not really necessary to reopen the connections on each command exectution. It's better to open the minimum number of them once and reuse them later.

Reply:
Does it mean that if the connection is dead, daemon will be unable to detect it until restart?

On top of that, it seems unclear why do we use something called  'statuscache' to provide some connection/folder. I think that we need to alter its name to represent the actual purpose, like `MultiStorageProvider` or something like that.

Reply:
> Does it mean that if the connection is dead, daemon will be unable to detect it until restart?

Hm, yeah, you're right. So, unfortunately, it means we need to explicitly close it somehow.